docs/plugins/convert.rst

```diff
@@ -93,6 +99,22 @@ file. The available options are:
 - **threads**: The number of threads to use for parallel encoding.
   By default, the plugin will detect the number of processors available and use
   them all.
+- **link**: By default, files that do not need to be transcoded will be copied
+  to their destination. This option creates symbolic links instead. Note that
+  options such as ``embed`` that modify the output files after the transcoding
+  step will cause the original files to be modified as well if ``link`` is
+  enabled. For this reason, album-art embedding is disabled
+  for files that are linked.
+  Default: ``false``.
+- **hardlink**: By default, files that do not need to be transcoded will be
+  copied to their destination. This option creates hard links instead. Note that
+  options such as ``embed`` that modify the output files after the transcoding
+  step will cause the original files to be modified as well if ``hardlink`` is
+  enabled. For this reason, album-art embedding is disabled
+  for files that are linked.
+  This option overrides ``link``. Only works when converting to a directory
+  on the same filesystem as the library.
+  Default: ``false``.
```

Comment:
No need to copy n' paste approximately the same description for both options—just say that the second one works similarly to the first, but it hard-links files instead.